Ui/CustomWidgets/CustomWidgets.py

```diff
@@ -0,0 +1,65 @@
+"""
+    This file converts custom widgets` .ui files to QWidget classes.
+    Nothing should be changed here, unless new files are added
+
+    Import this file as:
+
+    from CustomWidgets.CustomWidgets import *
+
+"""
+
+from PyQt5 import uic, QtWidgets
+
+
+class CreateConicalWidget(QtWidgets.QWidget):
+    def __init__(self):
+        super(CreateConicalWidget, self).__init__()
+        uic.loadUi("CustomWidgets\CreateConicalWidget.ui", self)
+
+
+class CreateCurveWidget(QtWidgets.QWidget):
+    def __init__(self):
+        super(CreateCurveWidget, self).__init__()
+        uic.loadUi("CustomWidgets\CreateCurveWidget.ui", self)
+
+
+class CreateCylindricalWidget(QtWidgets.QWidget):
+    def __init__(self):
+        super(CreateCylindricalWidget, self).__init__()
+        uic.loadUi("CustomWidgets\CreateCylindricalWidget.ui", self)
+
+
+class CreateLineWidget(QtWidgets.QWidget):
+    def __init__(self):
+        super(CreateLineWidget, self).__init__()
+        uic.loadUi("CustomWidgets\CreateLineWidget.ui", self)
```

Comment:
Write makefile to generate python code. Instead of doing it dynamicly here

Reply:
Why

Reply:
1. This pathes doesn't work for linux
2. We can only get buttons by string name
3. It is not possible modify any code without qteditor